test/data/compile-app-settings/invalid-purging-rules/project/purging.js

```diff
@@ -0,0 +1,6 @@
+function(userCtx, contact, reports) {
+  const old = Date.now() - (1000 * 60 * 60 * 24 * 365);
+
+  return reports
+    .filter(r => r.reported_date < old)
+    .map(r => r_.id);
```

Comment:
Add a comment to make it obvious why this is invalid I reckon. I missed it on my first pass.

Reply:
Perfect! Proving even more this check is useful :P

(I added a comment)